docs/src/main/asciidoc/infinispan-client-reference.adoc

```diff
@@ -824,12 +824,33 @@ They cannot be used with unbounded near cache: maximum number of entries must be
 
 Encryption at this point requires additional steps to get working.
 
-The first step is to configure the `hotrod-client.properties` file to point to your truststore
+The first step is to configure the `application.properties` file to point to your truststore
 and/or keystore. This is further detailed https://infinispan.org/docs/stable/titles/hotrod_java/hotrod_java.html#hotrod_encryption[here].
 
 The Infinispan Client extension enables SSL/TLS by default. You can read more about this
 at xref:native-and-ssl.adoc[Using SSL With Native Executables].
 
+== SSL Host Name Validation
+
+To prevent MITM attacks, when SSL is enabled, SSL host name validation is enabled by default in Infinispan.
+In this case, configuring the SNI host name is mandatory to start the client.
+
+[source,properties]
+----
+quarkus.infinispan-client.sni-host-name=localhost <1>
+----
+<1> Sets the SNI host name
+
+This behaviour can be changed by disabling the validation.
+
+[source,properties]
+----
+quarkus.infinispan-client.ssl-host-name-validation=false <1>
+----
+
```

Comment:
There is one extra line, but I can't see documentation preview, so maybe it won't cause any issues. It's fine if you are sure it will convert correctly.

Reply:
IIRC, it renders OK but I force pushed the change to be on the safe side as the doc is pushed to multiple inconsistent tools downstream.